doc.go

```diff
@@ -22,8 +22,183 @@ instance:
         })
     }
 
+Assertions
+
+An assertion looks like this, where qt.Equals could be replaced by any
+available checker. If the assertion fails, the underlying Fatal method is
+called to describe the error and abort the test.
+
+    c.Assert(someValue, qt.Equals, wantValue)
+
+If you don’t want to abort on failure, use Check instead, which calls Error
+instead of Fatal:
+
+    c.Check(someValue, qt.Equals, wantValue)
+
 The library provides some base checkers like Equals, DeepEquals, Matches,
 ErrorMatches, IsNil and others. More can be added by implementing the Checker
-interface.
+interface. Here is a list of checkers already included in the package.
+
+Equals
+
+Equals is a Checker checking equality of two comparable values.
+For instance:
+
+    c.Assert(answer, qt.Equals, 42)
+
+Note that the following will fail:
+
+    c.Assert((*sometype)(nil), qt.Equals, nil)
+
+Use the IsNil checker below for this kind of nil check.
+
+CmpEquals
```

Comment:
Maybe put this after DeepEquals, as that's much more common to use?
(alternatively you could sort them all alphabetically)

Reply:
Done.